index.js

```diff
@@ -145,44 +145,42 @@ else if (program.hub) {
         helpers.logError(error);
     });
 }
-else if(program.refreshAuthToken){
+else if (program.refreshAuthToken) {
     console.log("------ Refreshing oAuth token for hub %j".header, program.refreshAuthToken);
     var onboardingCli = new OnboardingCli();
-    onboardingCli.loadTranslatorAndGetOnboardingAnswers(program.refreshAuthToken).then(answers => {
-        var fileName = helpers.createOnboardingFileName(program.refreshAuthToken);
-        helpers.readFile(fileName, "Please complete onboarding -o").then(data => { 
+    var fileName = helpers.createOnboardingFileName(program.refreshAuthToken);
+    helpers.readFile(fileName, "Please complete onboarding -o").then(data => {
         let configInfo = JSON.parse(data);
         let authInfo = configInfo.authInfo;
-        translatorCli.getProperty(program.refreshAuthToken, authInfo, 'refreshAuthToken', answers).then(refreshedInfo => {
-            helpers.logObject(refreshedInfo);
-            console.log("------ Saving refreshed onboardingInfo to: " + fileName); 
-            let configData = helpers.createConfigData(configInfo.translator, configInfo.translatorPackageName, refreshedInfo);
-            let refreshedData = JSON.stringify(configData);
-            fs.writeFile(fileName, refreshedData, function (err) {
-                if (err) {
-                    console.log(err);
-                    return console.log(err);
-                }
-            console.log("Saved!");
-        });
+        onboardingCli.loadTranslatorAndGetOnboardingAnswers(configInfo.translatorPackageName).then(answers => {
+            translatorCli.getProperty(configInfo.translatorPackageName, authInfo, 'refreshAuthToken', answers).then(refreshedInfo => {
+                helpers.logObject(refreshedInfo);
+                console.log("------ Saving refreshed onboardingInfo to: " + fileName);
```

Comment:
There's a lot of legacy logging code in here that should be cleaned up.  I'd prefer to do that as it's own change.

---
In reply to: [103610773](https://github.com/openT2T/opent2t-cli/pull/24#discussion_r103610773) [](ancestors = 103610773)